crates/batch-prover/src/runner.rs

```diff
@@ -388,6 +388,7 @@ where
             .storage_manager
             .create_storage_on_l2_height(l2_height)?;
 
+        let mut signed_soft_confirmation: SignedSoftConfirmation = soft_confirmation.clone().into();
```

Comment:
just clone txs?

Reply:
Later we need signed_soft_confirmation too. 5 lines below